tests/utils/minishift_latest_version.py

```diff
@@ -3,8 +3,12 @@
 import requests
 import re
 import os
+import platform
 
-OS = os.getenv('OS', 'linux')
+distributionInfo = platform.platform()
+
+matchObj = re.match( r'linux|darwin|windows', distributionInfo, re.I)
```

Comment:
Also now this need to be changed since our tests, not python dependent anymore but bash scripts so windows platform shouldn't in the picture now.